Jenkinsfile

```diff
@@ -4,6 +4,7 @@ pipeline {
     agent any
```

Comment:
Probably in future it might make sense to do that by github actions.

Reply:
yes, if there is a need for any devops there should be some example, not to provide every single implementation